Training/view/frontend/templates/account/dashboard/credithold.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+/**
+ * @var Codifi\Training\Block\Account\Dashboard\CreditHold $block
+ */
+?>
+<?php if (
+    $block->isCreditHoldConfigEnabled() &&
+    $block->getCustomerAttr() &&
+    !$block->getFlag()
+): ?>
+    <div class="confirmation-modal-content">
+        <p><?php
+            echo $block->getMessage();
+            $block->setFlag();
```

Comment:
Actually, you can move all logic to `getFlag`. Check flag value. If it's true, don't show popup, else set flag value to the customer session and show popup.